cloudwatch/src/lambda_function.py

```diff
@@ -82,6 +86,15 @@ def _enrich_logs_data(aws_logs_data, context):
     try:
         aws_logs_data['function_version'] = context.function_version
         aws_logs_data['invoked_function_arn'] = context.invoked_function_arn
+
+        # If ENRICH has value, add the properties
+        if os.environ['ENRICH']:
+            data_to_enrich = dict()
+            properties_to_enrich = os.environ['ENRICH'].split(";")
+            for property_to_enrich in properties_to_enrich:
+                property_key_value = property_to_enrich.split("=")
+                data_to_enrich[property_key_value[0]] = property_key_value[1]
```

Comment:
And here you can just do:
```suggestion
               aws_logs_data[property_key_value[0]] = property_key_value[1]
```